css/index.css

```diff
@@ -0,0 +1,106 @@
+:root {
+  --top-header-height: 48px;
+  --default-font-family: 'inter', -apple-system, 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Open Sans', 'Helvetica Neue', sans-serif;
+  --theme-green-color: #36b37e;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+body * {
+  font-family: var(--default-font-family);
+}
+
+.top-header {
+  display: flex;
```

Comment:
- [ ] Congratulations, this was a good idea to create variables to avoid repetition 👏 , but do you know what? we can still improve this code and make it even better, I can see that you have used the property `display: flex;`  at multiple places in your CSS file, we can avoid this repetition by using **DRY**(Don't repeat yourself), but how?. Well, we can extract that property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

Reply:
Hello,

Thank you a lot for this insightful review. I will make another pull soon, and post-implementation of the suggested guides.

- HappyCoding 😃

Reply:
Hello, once more  @barackm,

I have successfully managed to make the changes.

Thank you once more. Hopefully, I get it right on the next review.